.github/workflows/conformance-runtime.yaml

```diff
@@ -267,7 +267,7 @@ jobs:
           test-name: runtime-tests
           install-dependencies: true
           # renovate: datasource=docker depName=quay.io/lvh-images/kind
-          image-version: "6.12-20241218.004849@sha256:906f475b884dc24d773bd8277d727dda720c2587f8a8d4b6fbff2fe050fa7025"
+          image-version: "6.12-20250116.014736@sha256:db83e562f90f8e6a3d3e996bd6a408ae2d70580c3e1b6077ee79231416a8f9af"
```

Comment:
@julianwiedmann it seems this kernel version has a regression that's causing our runtime tests to fail. Similar thing is happening in https://github.com/cilium/cilium/pull/37099

Reply:
you mean https://github.com/cilium/cilium/pull/37063 right? Yep saw that, but didn't look further. I'm somewhat suspicious that it's a regression in LVH base image ... the bpf-next regression happened somewhere during last week, it's unlikely that such a regression would have already trickled down into a kernel patch release.

Reply:
hmm I thought it was a stable branch, then it's likely the #37063. On main branch for example, there isn't a failure similar to the ones we are seeing with this PR. https://github.com/cilium/cilium/actions/workflows/conformance-runtime.yaml?query=branch%3Amain